src/deploy-actions.js

```diff
@@ -54,6 +55,11 @@ async function deployActions (config, deployConfig = {}, logFunc) {
   ) {
     throw new Error(`missing files in ${utils._relApp(config.root, dist)}, maybe you forgot to build your actions ?`)
   }
+  /// c. options param missing actionCode property
+  let optionsParam = options
```

Comment:
nitpick, `optionsParam = options`  will allow modifications on the `options` input object downstream, which might be a source for bugs in the future. maybe do something like `const optionsParm = { actionCode: options.actionCode !== false }` instead

Reply:
Sure @moritzraho . Thanks for pointing that out. I have made `optionsParam` as a `const`